modules/sno-du-subscribing-to-the-operators-needed-for-platform-configuration.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-subscribing-to-the-operators-needed-for-platform-configuration_{context}"]
+= Subscribing to the Operators needed for platform configuration
+
+The subscription provides the location to download the Operators needed for platform configuration.
+Configure the subscription as follows:
+
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: cluster-logging
+    namespace: openshift-logging
+spec:
+    channel: "5.0" <1>
+    name: cluster-logging
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: local-storage-operator
+    namespace: openshift-local-storage
+spec:
+    channel: "4.8" <2>
```

Comment:
Consider updating these to a more recent release.

Reply:
Isn't this release intended only for 4.9?

Reply:
The version in this channel should reflect the release covered by the documentation. Currently this reads 4.8. If these are 4.9 docs it should read 4.9.